automation_tools/satellite6/upgrade/__init__.py

```diff
@@ -109,6 +109,9 @@ def satellite6_upgrade(admin_password=None):
         # all the services before upgrade
         run('katello-service stop')
         run('service-wait mongod start')
+    # Verifying impact of BZ #1357655 on upgrade
+    if os.environ.get('RHEV_SATELLITE') or os.environ.get('SAT_HOST'):
```

Comment:
this check is only for automation. If someone try to use upgrade installer job for upgrading their own satellite then this would be skipped.